here/traffic/sql/create_tables.sql

```diff
@@ -1,73 +1,53 @@
-/*DROP TABLE IF EXISTS here.ta CASCADE;
-
-CREATE TABLE here.ta
+-- Parent table structure
+CREATE TABLE IF NOT EXISTS here.ta
 (
-  link_dir text NOT NULL,
-  tx timestamp without time zone NOT NULL,
-  epoch_min integer NOT NULL,
-  length int,
-  mean numeric(4,1) NOT NULL,
-  stddev numeric(4,1) NOT NULL,
-  min_spd integer NOT NULL,
-  max_spd integer NOT NULL,
-  confidence integer NOT NULL,
-  pct_5 int NOT NULL,
-pct_10 int NOT NULL,
-pct_15 int NOT NULL,
-pct_20 int NOT NULL,
-pct_25 int NOT NULL,
-pct_30 int NOT NULL,
-pct_35 int NOT NULL,
-pct_40 int NOT NULL,
-pct_45 int NOT NULL,
-pct_50 int NOT NULL,
-pct_55 int NOT NULL,
-pct_60 int NOT NULL,
-pct_65 int NOT NULL,
-pct_70 int NOT NULL,
-pct_75 int NOT NULL,
-pct_80 int NOT NULL,
-pct_85 int NOT NULL,
-pct_90 int NOT NULL,
-pct_95 int NOT NULL
-)
-WITH (
-  OIDS=FALSE
-);
-ALTER TABLE here.ta
-  OWNER TO here_admins;
-*/
+    link_dir text NOT NULL,
+	dt date NOT NULL,
+	tod time without time zone NOT NULL,
+	tx timestamp without time zone NOT NULL, 
+    length integer,
+    mean numeric(4,1) NOT NULL,
+    stddev numeric(4,1) NOT NULL,
+    min_spd integer NOT NULL,
+    max_spd integer NOT NULL,
+    pct_50 integer NOT NULL,
+    pct_85 integer NOT NULL,
+    confidence integer NOT NULL,
+	sample_size integer
+) PARTITION BY RANGE (dt);
+
+ALTER TABLE here.ta OWNER TO here_admins;
+GRANT SELECT ON TABLE here_staging.ta TO bdit_humans;
+GRANT INSERT, UPDATE, SELECT, TRIGGER ON TABLE here_staging.ta TO here_bot;
+
+-- Simple view structure 
+CREATE VIEW here.ta_view AS
+ 
+SELECT  link_dir, 
+        tx, 
+        NULL:integer as epoch_min, 
+        ength, 
+        sample_size, 
```

Comment:
flagging that sample size is not in the same location as table. Confirm this is correct location

Reply:
yup! confirming that this is the correct column order for the input csv.gz to copy to. Then the trigger reorganize the columns to fit the here.ta table column order.